malloc/malloc.c

```diff
@@ -26,6 +30,12 @@ char *string_dup(char *src)
 */
 void *mem_copy(void *dest, const void *src, int n)
 {
+    char *src = (char*)src;
+    int size = sizeof(src);
+    char *dest = malloc(size);
+    for (int i = 0; i < size/sizeof(char); i++) {
+        *(dest + i) = src[i];
+    }
```

Comment:
If you look at `main` function, you will find that the argument for the `dest` parameter is already a contiguous block of memory, so there is no need to in this function.

Reply:
You're properly handling the `dest` parameter. Please resolve these comments!